src/sidebar/host-config.js

```diff
@@ -50,13 +56,33 @@ export default function hostPageConfig(window) {
     'usernameUrl',
   ];
 
+  const coercions = {
```

Comment:
I'd suggest adding a comment here explaining:

1) Why we are coercing values (both the use case where "Via" passes through values as strings and where a publisher accidentally provides the wrong type)
2) A note about why only these two specific params are handled at the moment, but that there is an intent to expand this to other params as well.